README.md

```diff
@@ -600,3 +709,9 @@ and move philosophical, academic or otherwise unrelated technical
 discussions to an alternate forum.
 * Seek to understand the perspective of the author.
 * Sign off the final review with a thumbs up or some other positive remark.
+
+## Reproducibility guidelines
+
+Reproducibility is an important aspect of scientific research. As a support organisation directly involved in research we have a responsibility
+ to make sure that our work is reproducible. Please refer to the NBIS [Reproducibility guidelines](https://github.com/NBISweden/Reproducibility-Guidelines) 
+ for more information about this matter.
```

Comment:
@jbygdell Do you think this section and the referenced Reproducibility guidelines covers your thoughts in #18 ?